doc/usage/extensions/linkcode.rst

```diff
@@ -48,4 +55,29 @@ Configuration
           if not info['module']:
               return None
           filename = info['module'].replace('.', '/')
-          return "https://somesite/sourcerepo/%s.py" % filename
+          line_anchor = ""
+          if info['start_line'] >= 0:
+            line_anchor = "#L%s" % info['start_line']
```

Comment:
Any plans to add the `end_line` to the line anchor?  It seems that the format `#L10-L23` is supported by a few web-based code viewers (checked: [GitHub](https://github.com/sphinx-doc/sphinx/blob/faa33a53a389f6f8bc1f6ae97d6015fa92393c4a/README.rst?plain=1#L1-L3), [GitLab](https://gitlab.com/gitlab-org/gitlab/-/blob/23451dc906d3d43a2aca49ec606d01d39363ec4e/.gitlab-ci.yml#L21-24), [Gitea](https://gitea.com/gitea/tea/src/branch/main/Makefile#L120-L121), [Debsources](https://sources.debian.org/src/sphinx/7.2.6-4/README.rst/#L1-L3)).